assets/queries/terraform/kubernetes/volume_mount_with_os_directory_write_permissions/query.rego

```diff
@@ -0,0 +1,190 @@
+package Cx
+
+types := {"init_container", "container"}
+
+CxPolicy[result] {
+	resource := input.document[i].resource[resourceType]
+
+	spec := resource[name].spec
+	containers := spec[types[x]]
+
+	is_array(containers) == true
+
+	volumeMounts := containers[y].volume_mount
+	is_object(volumeMounts) == true
+	is_OS_Dir(volumeMounts)
```

Comment:
I think we should try to stick to only one code style, either use CammelCase or snake_case, we should choose one.... it seems that rego already works with snake_case. Maybe we should change this and other functions to snake_case:
```suggestion
	is_os_dir(volumeMounts)
```